### ISSUE TITLE
SSL certificate on demo page expired

### ISSUE THREAD

**Issue by oshiteku** [stumpapp/stump#420]
**Describe the bug**

https://demo.stumpapp.dev/ could not be successfully viewed because the SSL certificate has expired.

**Screenshots**

![image](https://github.com/user-attachments/assets/6c9532d8-85f4-46d3-80e8-e49aa56886c6)

**Desktop (please complete the following information):**

- OS: Ubuntu 24.04, Windows 11

**Comment by aaronleopold** [comment]
Thanks for reporting this! I’ll fix it once I’m by a computer again (probably tomorrow). Sorry you can’t try it out in the meantime, but I’ll close this issue once it’s back up.

**Comment by aaronleopold** [comment]
Yeah so it appears the auto-renewing cert logic I had configured broke from the Google-to-Squarespace migration 😩 I don't expect to have time to fix this for bit, so the demo will be down in the meantime.

**Comment by JoaGamo** [comment]
Hey @aaronleopold ! Since this issue is a year old, please reconsider to fix the demo ;)

**Comment by aaronleopold** [comment]
Hey @JoaGamo, I appreciate the reminder! 

The demo is something I paid to host out of pocket since I didn't want to expose something publicly from my own homelab to the world, that combined with the fact that its just another thing I'd have to either manually maintain or build out infra to automate honestly makes it a very low priority for me. The thing consuming the most of my (already limited) time is https://github.com/stumpapp/stump/discussions/634, which I'm hopeful to get into `experimental` in the next week or so, and after that will likely be a lot of bug fixes and feature development.

I'd welcome contributions to the configs behind the demo (currently behind a private repo, but there isn't anything sensitive really) if that is an area of interest, but I unfortunately won't have the time to work on it for the foreseeable future. I'm also trying to save money 🤑 so would want to look for free hosting, or at least cheaper than the droplet I was using

**Comment by JoaGamo** [comment]
Thank you very much for answering so quickly!
Consider using things like Oracle Cloud Free Tier, it fits the purpose of a demo server while being free. As it will be mostly read-only I assume, no important data can be randomly taken down by Oracle.
Google Cloud had a free VPS tier with their 'nano' VPS, not sure if they are still around now
